files/zh-cn/web/http/cookies/index.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: HTTP cookies
+title: HTTP Cookies
```

Comment:
I prefer `Cookie` not `Cookies`. On the other hand, the documentation needs to be updated.